### DISCUSSION TITLE
How do you mark a Homebrew package as a release candidate?

### DISCUSSION THREAD

**Discussion by sr010** [Homebrew/discussions#2363] [Getting started]
Say a homebrew package was released for internal testing purposes. Basically a release candidate. How do I mark the homebrew package as a release candidate so that it doesn't get installed when a user tries to install the latest package? I need the latest stable package to get installed.

**Comment by gromgit** [comment]
The simplest way is to release the RCs as [versioned formulae](https://docs.brew.sh/Versions), so `mypkg@rc` for RCs vs `mypkg` for release versions. Each formula should probably `conflicts_with` the other, so you can't install both at once. This has the benefit that `brew upgrade` does the right thing for both new versions and new RCs.
